clustered/server/build.gradle

```diff
@@ -35,4 +35,5 @@ dependencies {
   providedImplementation "org.terracotta:runnel:$terracottaPlatformVersion"
 
   testImplementation project(':clustered:test-utils')
+  testImplementation ("org.terracotta:passthrough-server:$terracottaPassthroughTestingVersion")
```

Comment:
Why does this need to be added?

Reply:
Offheap is now instrumented with tripwire which is provided by the server.  There are some unit tests that instantiate OffheapProvider without a server so passthrough is added to provide the required libs.  We could just add tripwire but that is one more dependency to manage.

Reply:
Okay.